src/components/gamer/GamerSearchInput.tsx

```diff
@@ -0,0 +1,57 @@
+import React, {useState} from 'react';
+
+import {Box, Container} from './../SimpleComponents';
+
+import TypeService from '../../services/TypeService';
+
+import GamerLink from './GamerLink';
+
+import GamerCardList from './GamerCardList';
+import {GamerList, Gamer} from './GamerTypes';
+import {Input} from '../SmartComponents';
+
+//===---==--=-=--==---===----===---==--=-=--==---===----//
+
+type GamerSearchInputProps = {
+    size: 'xl' | 'lg' | 'md' | 'sm',
+    baseUrl?: string,
+    focus?: boolean,
+    mode?: null | 'condensed',
+    onGamerClick?: (gamer) => void
+}
+
+export default function GamerSearchInput(props: GamerSearchInputProps) {
```

Comment:
Converted the gamer search input to it's own component so we can reuse it and use it on the dashboard for the main gamer register feature